common/logging/EventAuditLogger.php

```diff
@@ -315,7 +315,7 @@ public function getEvents($params)
         } else {
             // do the query
             $sqlBindArray = array();
-            $sql = "SELECT $cols FROM log WHERE date >= ? AND date <= ?";
+            $sql = "SELECT " . escape_sql_column_name(process_cols_escape($cols), array('log')) . " FROM log WHERE date >= ? AND date <= ?";
```

Comment:
Check out the default $cols on line 218. Note the DISTINCT keyword will break our function. Recommend reverting change above and noting this as an exception. We can work on way to make this work in the future.